src/utils/internal/getCallFrame.ts

```diff
@@ -2,7 +2,7 @@
  * Return the stack trace frame of a function's invocation.
  */
 export function getCallFrame() {
-  const frames: string[] = (new Error().stack as string).split('\n')
+  const frames: string[] = ((new Error().stack || '') as string).split('\n')
```

Comment:
Full disclosure, I haven't done much TS at all-- mostly only reading so if this needs to change or there is a better way to write it I'm all ears.

Reply:
The only other option would be `??` it, but it doesn't really matter here and the `||` makes sense. I think for future readability, we should add a comment above this explaining why this is here (ex: < IE11 returns `undefined` instead of a string).

Reply:
Thanks @msutkowski, added in 1b3873afba671d3fe707f6e019264a1c6fa931f8.

Reply:
We should be careful about `??`, since that wouldn't handle `false`, which is still an unexpected value that mustn't be treated as an error's stack trace. I think `||` is the way to go here.